.travis.yml

```diff
@@ -1,5 +1,54 @@
 language: node_js
 node_js:
-  - "0.10"
+- '0.10'
 before_install: npm install -g grunt-cli
-install: npm install
+install:
```

Comment:
Shouldn't need to change `install` stage, nor include `grunt test` in any way.
Instead, use the `before_deploy` stage for the `grunt dist` and `grunt codedeploy` tasks
See: https://github.com/cfpb/hmda-pilot/blob/milestone3/.travis.yml#L7